### DISCUSSION TITLE
remote: Repository not found.

### DISCUSSION THREAD

**Discussion by indiherbsAyurvedah** [community/community#80892] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

$ git push -u origin main
remote: Repository not found.
fatal: repository 'https://github.com/indiherbsAyurvedah/ihbend.git/' not found

**Comment by marcel-rayane** [comment]
Hi,
I have the same problem. Have you found a solution ?
Tks
